### PR TITLE
Initial commit for the LazoApp documentation

### PR DESCRIPTION
**Pull request by subramgr**

### CODE REVIEW

File: docs/LazoApp.md [hunk 1]
```diff
@@ -0,0 +1,554 @@
+`LazoApp` class provides a common set of apis that can be used at both client and server side. Your application has to extend `LazoApp` in `application.js` file. This is where you load environment specific configuration, set application level properties, and define application level CSS and JavaScript. Please check the Lazo Application Strucure for more information [LazoApplicationStrucutre](LazoApplicationStrucutre.md)
```
Comment (jstrimpel) [inline docs/LazoApp.md]: Please remove "define application level CSS and JavaScript" or add a section that covers `app.json`, https://github.com/lazojs/lazo/wiki/Configuration#appjson

File: docs/LazoApp.md [hunk 2]
```diff
@@ -0,0 +1,554 @@
...
+
+Example:
+```javascript
+define(['lazoApp'], function (LazoApp) {
+
+    'use strict';
+
+    return LazoApp.extend({
+
+        css: ['/app/client/app.css'],
```
Comment (jstrimpel) [inline docs/LazoApp.md]: `css` and `js` are now handled by `app.json`

File: docs/LazoApp.md (hunk 1 above)
Comment (subramgr) [inline docs/LazoApp.md]: Removed. Will cover app.json

File: docs/LazoApp.md (hunk 2 above)
Comment (subramgr) [inline docs/LazoApp.md]: Removed, will cover configuration separately

**Pull request merged**